btsieve/src/ethereum/pattern_matching.rs

```diff
@@ -5,7 +5,10 @@ use ethereum_support::{
 };
 
 #[derive(Clone, Default, Debug, Eq, PartialEq)]
-pub struct TransactionQuery {
+// If the field is set to Some(foo) then only transactions matching foo are
```

Comment:
Should be a rust-doc comment (triple slash) :)

Reply:
Sweet - done.